tensorflow/tensorflow_plugin.cc

```diff
@@ -854,54 +859,60 @@ struct TensorflowGraphBase: public Function {
             }
             
             switch (type) {
-            case tensorflow::DT_FLOAT: {
```

Comment:
sorry, had issue with an unmatched brace, and the indentation change didnt appear in git diff.